SPECS/LICENSES-AND-NOTICES/data/licenses.json

```diff
@@ -2208,7 +2208,7 @@
                 "lsb-release",
                 "ltp",
                 "lttng-consume",
-                "mariner-release",
+                "azurelinux-release",
```

Comment:
Please put in alphabetical order or just run the `license_map.py` script, which would organize this for you:
```
pushd [repo_root]

toolkit/scripts/license_map.py --no_check --update --remove_missing SPECS/LICENSES-AND-NOTICES/data/licenses.json SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md SPECS SPECS-EXTENDED SPECS-SIGNED
```

Reply:
I ran it without `--remove_missing`, otherwise it was generating massive changes to the entire `licenses.json` file, maybe it is very outdated?